src/main/java/core/basesyntax/service/FruitFIleWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FruitFIleWriterImpl implements FruitFileWriter {
```

Comment:
```suggestion
public class FileWriterImpl implements FileWriter {
```
we can actually write any content to any file with this service